src/DssSpell.t.sol

```diff
@@ -95,7 +95,21 @@ contract DssSpellTest is DSTest, DSMath {
     FlipperMomAbstract   flipMom = FlipperMomAbstract( addr.addr("FLIPPER_MOM"));
     DssAutoLineAbstract autoLine = DssAutoLineAbstract(addr.addr("MCD_IAM_AUTO_LINE"));
 
-    // Specific for this spell
+    // UNIV2DAIUSDC-A specific
+    DSTokenAbstract     lpDaiUsdc = DSTokenAbstract(addr.addr("UNIV2DAIUSDC"));
+    GemJoinAbstract lpJoinDaiUsdc = GemJoinAbstract(addr.addr("MCD_JOIN_UNIV2DAIUSDC_A"));
+    FlipAbstract    lpFlipDaiUsdc = FlipAbstract(   addr.addr("MCD_FLIP_UNIV2DAIUSDC_A"));
+    LPOsmAbstract    lpPipDaiUsdc = LPOsmAbstract(  addr.addr("PIP_UNIV2DAIUSDC"));
+    MedianAbstract    orb0DaiUsdc = MedianAbstract( lpPipDaiUsdc.orb0());
+    MedianAbstract    orb1DaiUsdc = MedianAbstract( lpPipDaiUsdc.orb1());
+
+    // UNIV2ETHUSDT-A specific
+    DSTokenAbstract     lpEthUsdt = DSTokenAbstract(addr.addr("UNIV2ETHUSDT"));
+    GemJoinAbstract lpJoinEthUsdt = GemJoinAbstract(addr.addr("MCD_JOIN_UNIV2ETHUSDT_A"));
+    FlipAbstract    lpFlipEthUsdt = FlipAbstract(   addr.addr("MCD_FLIP_UNIV2ETHUSDT_A"));
+    LPOsmAbstract    lpPipEthUsdt = LPOsmAbstract(  addr.addr("PIP_UNIV2ETHUSDT"));
+    MedianAbstract    orb0EthUsdt = MedianAbstract( lpPipEthUsdt.orb0());
+    MedianAbstract    orb1EthUsdt = MedianAbstract( lpPipEthUsdt.orb1());
```

Comment:
Do we need all of these if they were added in the addresses?

Reply:
These are just setting them to their abstracts so they don't have to be cast every time in the integration test.

Reply:
:facepalm: